client/src/pages/Dashboard.js

```diff
@@ -0,0 +1,165 @@
+import React, { useState } from "react";
+import { withRouter } from "react-router-dom";
+import {
+  Grid,
+  Dialog,
+  Hidden,
+  Paper,
+  Button,
+  MenuList,
+  MenuItem,
+} from "@material-ui/core";
+import ArrowBackIcon from "@material-ui/icons/ArrowBack";
+import { makeStyles } from "@material-ui/core";
+import EditProfile from "../components/dashboard/EditProfile";
+import ProfilePhoto from "../components/dashboard/ProfilePhoto";
+import Availability from "../components/dashboard/Availability";
+import Payment from "../components/dashboard/Payment";
+import Security from "../components/dashboard/Security";
+import Settings from "../components/dashboard/Settings";
+
+const useStyles = makeStyles(theme => ({
+  root: { flexgrow: 1 },
+  menuBox: {
+    "&:focus": {
+      outline: "none",
+    },
+  },
+  menuItem: {
+    fontSize: "20px",
+    margin: "24px auto",
+    width: "fit-content",
+  },
+  logout: {
+    color: "red",
+  },
+  BackButton: {
+    height: "60px",
+    fontSize: "24px",
+    borderRadius: "0",
+  },
+}));
+
+function Dashboard() {
+  const [open, setOpen] = useState(false);
+  const [pageShown, setPageShown] = useState(1);
+  const classes = useStyles();
+
+  //rendering windows corresponding to menu options clicked
+  function renderSwitch(pageShown) {
+    switch (pageShown) {
```

Comment:
Would it make more sense to use Routes and Links instead of a Switch statement? Something like "/profile/{subpage}"

Reply:
It depends on if we want the URL change at every option or not. We might want it change though since we would make post request at every "page". I'll take a look at that